### PR TITLE
[feat] added avatar

### PR DESCRIPTION
**Pull request by omjadhav85**
Added avatar component
image

### CODE REVIEW

File: styles/utilities/variables.css [hunk 1]
```diff
@@ -0,0 +1,8 @@
+:root {
+  --primary-color: hsl(196, 77%, 55%);
```
Comment (prachi-sahani) [inline styles/utilities/variables.css]: try using consistent units for colors

**Review by prachi-sahani — COMMENTED** [review]
What is the significance of "stl" in your class names?

Everything else seems fine 👍🏻

File: styles/utilities/variables.css (hunk 1 above)
Comment (omjadhav85) [reply styles/utilities/variables.css]: Thanks!! Will change to similar hex codes. And for the "stl" suffix is for "SubtleUI" library name but i am planning ot remove it as its making the class names seem longer

**Review by piyushsh25 — COMMENTED** [review]
I'd suggest you use rem values for >10px. Else everything looks great.

File: styles/components/avatar.css [hunk 2]
```diff
@@ -0,0 +1,22 @@
+.avatar {
+  border-radius: 50%;
+  width: 70px;
+  height: 70px;
+  margin: 0.5rem;
```
Comment (bajajcodes) [inline styles/components/avatar.css:5]: I believe giving margin here might break it somewhere or may produce unexpected results.

Comment (omjadhav85) [reply styles/components/avatar.css:5]: I actually gave it so that some other content following it should not look too crowded or close

**Comment by omjadhav85** [comment]
> I'd suggest you use rem values for >10px. Else everything looks great.
Thanks Piyush! Made the changes in code.

File: styles/components/avatar.css (hunk 2 above)
Comment (bajajcodes) [reply styles/components/avatar.css:5]: Yeah that's also the case, later you may have to override margin here and there.

I hope you understood.

**Pull request closed without merging**